app/app_test.go

```diff
@@ -108,7 +108,11 @@ func TestPrecompilesAreBlockedAddrs(t *testing.T) {
 	// For now there are no exceptions, so this slice is empty.
 	var precompilesAbleToReceiveFunds []ethcommon.Address
 
-	availablePrecompiles := network.App.EvmKeeper.GetAvailablePrecompileAddrs()
+	hexAvailablePrecompiles := network.App.EvmKeeper.GetParams(network.GetContext()).ActiveStaticPrecompiles
+	availablePrecompiles := make([]ethcommon.Address, len(hexAvailablePrecompiles))
+	for i, precompile := range hexAvailablePrecompiles {
+		availablePrecompiles[i] = ethcommon.HexToAddress(precompile)
+	}
 	for _, precompileAddr := range availablePrecompiles {
```

Comment:
All these tests are not failing for the proper reason:
- One fails because the address cannot receive funds (p256 precompile)
- all others during the estimate gas.